student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(name, age, _parent_permission, classroom)
+    super(name, age, parent_permission: true)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In initialize method here you but the default values for the inherited variables the wrong way. So please fix that.
[Hint]: It's supposed to be like that:
```
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```